### ISSUE TITLE
Navigating to edge:// causes exceptions either at start up or when the control is disposed

### ISSUE THREAD

**Issue by dqwork** [MicrosoftEdge/WebView2Feedback#1057]
**Description**
Two I think related issues:

1. Navigation to edge:// initially presents a blank control, if you then close the form, an exception is thrown when disposing the web control 
Exception Message: The group or resource is not in the correct state to perform the requested operation. (Exception from HRESULT: 0x8007139F)
Stack Trace:    at Microsoft.Web.WebView2.Core.Raw.ICoreWebView2Controller.set_ParentWindow(IntPtr ParentWindow)
   at Microsoft.Web.WebView2.Core.CoreWebView2Controller.set_ParentWindow(IntPtr value)
   at Microsoft.Web.WebView2.WinForms.WebView2.WebView2_HandleDestroyed(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnHandleDestroyed(EventArgs e)
   at System.Windows.Forms.Control.WmDestroy(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at Microsoft.Web.WebView2.WinForms.WebView2.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

2. If the first page you navigate to after running and awaiting InitializeAsync is edge:// the control throws an exception during its 'start up'.
Exception Message: The group or resource is not in the correct state to perform the requested operation. (Exception from HRESULT: 0x8007139F)
Stack Trace:    at Microsoft.Web.WebView2.Core.Raw.ICoreWebView2Controller.MoveFocus(COREWEBVIEW2_MOVE_FOCUS_REASON reason)
   at Microsoft.Web.WebView2.Core.CoreWebView2Controller.MoveFocus(CoreWebView2MoveFocusReason reason)
   at Microsoft.Web.WebView2.WinForms.WebView2.OnGotFocus(EventArgs e)
   at System.Windows.Forms.Control.InvokeGotFocus(Control toInvoke, EventArgs e)
   at System.Windows.Forms.Control.WmSetFocus(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at Microsoft.Web.WebView2.WinForms.WebView2.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Version**
SDK: 1.0.774.44
Runtime: 89.0.774.48
Framework: Winforms .NET Framework 4.6.2
OS: Windows 10

**Repro Steps**
Basic winforms set up with a webview and a button, that after await InitialiseAsync Navigate to:
Scenario 1: some valid website and then on the button press navigates to edge:// - then close the form
Scenario 2: edge:// 

Code example:

```
using System;
using System.Diagnostics;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WebView2_Testing
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            webView.CoreWebView2InitializationCompleted += WebView_CoreWebView2Ready;
        }

        protected override async void OnLoad(EventArgs e)
        {
            await InitializeAsync();
            //  use this source to fail at startup
            //webView.Source = new Uri("edge://");
            webView.Source = new Uri("https://www.github.com");
            Debug.WriteLine(webView.CoreWebView2.Environment.BrowserVersionString);
            base.OnLoad(e);
        }
        private void WebView_CoreWebView2Ready(object sender, EventArgs e)
        {
        }

        private async Task InitializeAsync()
        {
            await webView.EnsureCoreWebView2Async(null);
            Debug.WriteLine(webView.CoreWebView2.Environment.BrowserVersionString);
        }

        private void refreshBtn_Click(object sender, EventArgs e)
        {
            webView.Source = new Uri("edge://");
        }
    }
}

```

**Screenshots**

**Additional context**
There has been various issues around disposal mentioned before and sometimes mentions a similar exception, however a lot of the time its mentioned to set the visible flag to false before disposal. For scenario 1 I tried that, it just threw the exception there instead. For scenario 2 (start up failing) not sure how that would work to be honest

**Comment by cgeier** [comment]
Some of the "edge://..." URLs are blocked, so not sure if this is a bug. The exception message needs improvement though.  See issue #604 and [WebView2 Release Notes](https://docs.microsoft.com/en-us/microsoft-edge/webview2/releasenotes#09579)

**Comment by dqwork** [comment]
I think it is a bug. I understand that some edge:// urls are blocked.. in fact I'm a big fan of that. The issue is my application crashing/logging errors, because some one navigated to 'edge://' and then closed the control, which throws the exception described above

**Comment by champnic** [comment]
Yes this is indeed a bug. We just put in a fix for #604, the issue cgeier linked above, and it should be available now in the latest Microsoft Edge Canary builds (90.0.815.0+). Let me know if that doesn't work and I can reopen this issue.
